Makefile

```diff
@@ -53,7 +53,7 @@ buildweb:
 	node --version
 	npm --version
 	rm -rf ${WEBROOT}/build
-	cd ${WEBROOT} && npm ci --ignore-scripts
+	cd ${WEBROOT} && npm ci
```

Comment:
with `--ignore-scripts` esbuild can't get the correct binary for linux and the build failed